docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -483,9 +483,9 @@ Current set of claims definitions:
 | sgx_config_svn             | uint16_t           | SGX           |Enclave configuration security version.                              |
 | sgx_isv_family_id          | uint8_t[16]        | SGX           |Enclave family ID.                                                   |
 | sgx_cpu_svn                | uint8_t[16]        | SGX           |Enclave CPU security version from sgx_report_t.                      |
-| sgx_pce_svn                | uint16_t           | SGX Remote    |PCE security version from quote.                                     |
-| sgx_qe_id                  | uint8_t[16]        | SGX Remote    |SGX Quoting Enclave ID, referred as QE_ID in SGX_ECDSA_QuoteGenReference_DCAP_API 3.1.4|
-| sgx_fmspc                  | uint8_t[6]         | SGX Remote    |SGX Family-Model-Stepping-Platform-Custom SKU from PCE certificate.  |
+| software_svn               | uint16_t           | Remote Only   |PCE security version from quote.                                     |
+| platform_id                | uint8_t[16]        | Remote Only   |Quoting platform ID. (referred as QE_ID in SGX_ECDSA_QuoteGenReference_DCAP_API 3.1.4)|
```

Comment:
1. The term "quote" is an SGX specific term, so "quoting platform ID" is still SGX specific (for example, it occurs nowhere in https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture or in the CCC whitepapers).  Before I can tell whether this claim is really SGX specific or not, can you tell me whether it maps to anything in the IETF RATS draft https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-04 ?   If it does not, do we have any other reason to believe it is not SGX specific?
2. SGX_ECDSA_QuoteGenReference_DCAP_API should be a clickable link.

Reply:
Suggest removing "Quoting" to make it generic.

Reply:
PCE is also SGX specific term

Reply:
> PCE is also SGX specific term

Good point.   If software_svn is supposed to be non-sgx specific then it should probably match what is in https://datatracker.ietf.org/doc/html/[draft-ietf-rats-eat-12](https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-12#section-3.10)#section-3.10 and contain a coswid version-scheme.   If the value of this claim for SGX is not a coswid version-scheme, then it would be better to keep the sgx-specific name and add a separate claim for the non-sgx specific one.

Reply:
Hi @dthaler, after the sig-arch meeting this was the conclusion we came to:
1. sgx_qe_id to be renamed to ueid (https://datatracker.ietf.org/doc/html/[draft-ietf-rats-eat-04](https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-04#section-3.4.1)#section-3.4.1)
2. sgx_fmspc to be renamed to hardware_model (https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat#section-3.7)
3. sgx_pce_svn be moved to sgx specific claims